include/entity.h

```diff
@@ -958,11 +958,12 @@ typedef struct {
 
 typedef struct {
     /* 0x7C */ s8 unk7C;
-    /* 0x7D */ s8 unk7D;
+    /* 0x7D */ u8 unk7D;
```

Comment:
This change is needed for the blue axe knight, and did not cause any issues with the green axe knight. If preferred I can pull these into separate extensions but I don't think it is necessary.